docs/adoc/SdkWizardPage.adoc

```diff
@@ -34,7 +34,7 @@ The context can be derived from a package selected in the Package Explorer or fr
 
 Source Folder:: The source folder of the Maven client module used for the creation of the page. The default value is the `src/main/java` folder in the Maven client module.
 Package:: The Java package that will contain the page class. The Scout SDK will try to guess the package name from the current context and derive matching package names for the Maven shared module.
-Name:: The name of the page class. According to Scout conventions the class name ends with the suffix `Page`.
+Name:: The name of the page class. According to Scout conventions the class name ends with the suffix `TablePage`.
```

Comment:
Partially correct: The suffix is "TablePage" (as suggested by the wizard) when the page is a subtype of `AbstractPageWithTable`. For `AbstractPageWithNodes` on the other hand, the suffix is "NodePage".
I'd suggest slightly rephrasing this as: "... ends with the suffic TablePage (for subclasses of AbstractPageWithTable) or NodePage (for AbstractPageWithNodes)"